background/lib/alchemy.ts

```diff
@@ -191,7 +191,11 @@ export async function getTokenBalances(
       // A hex value of 0x without any subsequent numbers generally means "no
       // value" (as opposed to 0) in Ethereum implementations, so filter it out
       // as effectively undefined.
-      .filter(({ tokenBalance }) => tokenBalance !== "0x")
+      .filter(
+        ({ tokenBalance }) =>
+          tokenBalance !==
+          "0x0000000000000000000000000000000000000000000000000000000000000000"
+      )
```

Comment:
Maybe something more like:
```suggestion
      .filter(
        ({ tokenBalance }) =>
          Number(tokenBalance) !== 0
      )
```

Reply:
> Some tokens on the list have "bigger" zeros 😉

My eyes are bleeding